core/src/test/java/io/questdb/std/str/PathTest.java

```diff
@@ -53,6 +54,25 @@ public void testConcatWithSlash() {
         TestUtils.assertEquals("xyz" + separator + "123", path.of("xyz/").concat("123").$());
     }
 
+    @Test
+    public void testDollarIdempotent() {
+        final CharSequence tableName = "table_name";
+        final AtomicInteger extendCount = new AtomicInteger();
+        try (Path path = new Path(0) {
+            @Override
+            void extend(int len) {
+                super.extend(len);
+                extendCount.incrementAndGet();
+            }
+        }) {
+            path.of(tableName).$();
+            for (int i = 0; i < 50_000; i++) {
```

Comment:
nit: you can only do it 2-3 times to prove the point, 50k iterations is an overkill, isn't it?

Reply:
changed it